### PR TITLE
[10.0][FIX][hr_holidays_settings] Make 'Leave Types' view accessible

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Fixes https://github.com/OCA/hr/issues/407.

cc: @feketemihai

### CODE REVIEW

File: hr_holidays_settings/views/hr_holidays_config_settings.xml [hunk 1]
```diff
@@ -32,4 +32,9 @@
               action="hr_holidays_config_settings_action"
               groups="base.group_system"/>
 
+    <menuitem id="menu_hr_holidays_status" name="Leave Types"
+          parent="hr_holidays.menu_hr_holidays_configuration" sequence="10"
+          action="open_view_holiday_status"
```
Comment (LoisRForgeFlow) [inline hr_holidays_settings/views/hr_holidays_config_settings.xml]: you need the complete external ID here I think: `hr_holiday.open_view_holiday_status` I guess but check it please

**Review by LoisRForgeFlow — APPROVED** [review]
Code review, LGTM :+1:

**Review by feketemihai — APPROVED** [review]
Not able to tested, something is wrong with runbot, it's redirecting to the OCB repo...

**Comment by MiquelRForgeFlow** [comment]
@feketemihai try here: [runbot](http://3310204-408-980446.runbot1.odoo-community.org/web/login).

**Review by feketemihai — APPROVED** [review]

**Comment by MiquelRForgeFlow** [comment]
@feketemihai will this be merged?

**Comment by feketemihai** [comment]
@mreficent The rules say that we still need one approval from HR maintainers, @OCA/human-resources-maintainers ?

**Comment by MiquelRForgeFlow** [comment]
Ok, then let's wait to one more approval for this fix

**Pull request merged**